### ISSUE TITLE
piecemeal tweaks to islandora_exhibits_browse_build_results 

### ISSUE THREAD

**Issue by wgilling** [simonhm/islandora_exhibits_browse#4]
Take these as you will... (sorry in advance on this one)

1. Within the islandora_exhibits_browse_build_results function, I believe that it needs to have fields set in the solrParams['fl'].  I modified the line:
  $solr_build->solrParams['fl'] = '';
to be:
  $solr_build->solrParams['fl'] = $fl_params;

2. I tested this with a collection of newspaper issues we have that contains over 6,500 issues.  The block was configured to point to this collection, yet the page timed out and gave a white screen with no html.  There was no error in the watchdog nor the apache log.  The page loaded fine when I manually updated the solrLimit to a number less than 1000.

3. We already had a menu route handler that pointed to the /exhibits which was easy enough to change by editing the Solr view we had... but it could be possible to add a path prefix to the configure for this that could be read within islandora_exhibits_browse_menu() when setting the $items[] value for the exhibits handler.

**Comment by simonhm** [comment]
Oh, thanks for your testing and suggestion ... 
1. Actually the $fl_params was supposed to pull out the list of fields we configure for headline, description, start date,... for each exhibit pages. Ppl can change this list of fields to match their site. 

![image](https://user-images.githubusercontent.com/6493383/37831121-599ab882-2e72-11e8-8518-6bebed5708ac.png)

For quick testing, I just put $solr_build->solrParams['fl'] = '';  ^_^ Yah sure, we will need to develop more at this point. 

2. I think there may be practical limits of browser performance, and more importantly, intelligibility. A timeline/slideshow with that many events (6500 issues) may be too much and your browser couldn't handle, could it ? 

3. Sorry, but I don't get really what you're talking about at this point. What is the benefit of adding this path prefix ?

Again, thank you for testing and pointing out what we need to improve for this module, Brian.

Simon.

**Comment by wgilling** [comment]
With respect to 3., this is probably asking too much -- it was just a collision of the same path we had already set up as a Solr View to exhibit some of our content.
![image](https://user-images.githubusercontent.com/19391126/37833579-0783b926-2e82-11e8-8324-3f04d8f0f622.png)

If the /admin/islandora/tools/exhibits_browse configuration page had a text field to set up the menu route prefix for these block views, the code in your islandora_exhibits.browse.module islandora_exhibits_browse_menu() could read that prefix and set up the configured path for the pages like this:
```
  $items[variable_get('islandora_exhibits_browse_page_prefix', 'exhibits')] = array(
    'page callback' => 'islandora_exhibits_browse_callback',
    'page arguments' => array(1),
    'access callback' => 'islandora_exhibits_browse_access_callback',
    'access arguments' => array(1),
    'type' => MENU_CALLBACK,
  );
```

Make sense?

I made a fork of the code and have a feature branch for this change and will be making a pull request after sending this message.
